scripts/referrer-user-count.ts

```diff
@@ -0,0 +1,74 @@
+import yargs from 'yargs'
+import { protocolFilters, supportedNetworkIds } from './consts'
+import { fetchReferralEvents, removeDuplicates } from './referrals'
+import { NetworkId, Protocol, protocols } from './types'
+import { writeFileSync } from 'fs'
+
+async function getArgs() {
+  const argv = await yargs
+    .env('')
+    .option('protocol', {
+      description: 'protocol that the referrals are for',
+      demandOption: true,
+      choices: protocols,
+    })
+    // This is coerced as a string array to prevent 0x... from being interpreted as a number
+    .option('referrerAddresses', {
```

Comment:
if you name it `foo-bar`, yargs automatically provides two properties: `['foo-bar'] and ['fooBar']`.  So use the dashed naming.